### PR TITLE
docs: using createStore to interact with atoms outside React 

### PR DESCRIPTION
**Pull request by ashuvssut**
## Related Issues or Discussions

- #263

## Summary

Added docs to explain how to interact with atoms outside React

This Codesandbox is used in the example: https://codesandbox.io/p/sandbox/practical-bardeen-2jvynm?file=%2Fsrc%2FApp.tsx%3A16%2C5

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: docs/guides/using-store-outside-react.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+[`createStore`](../core/store) provides a store interface that can be used to store your atoms. Using the store, you can access and mutate the state of your stored atoms from outside React.
+
+
+```jsx
+import { atom, useAtomValue, createStore, Provider } from "jotai";
+
+const timeAtom = atom(0);
+const store = createStore();
+
+store.set(timeAtom, store.get(timeAtom) + 1); // Update atom's value
```
Comment (dai-shi) [inline docs/guides/using-store-outside-react.mdx:23]: I think it's better to show function update in this case:
```suggestion
store.set(timeAtom, (prev) => prev + 1); // Update atom's value
```

File: docs/guides/using-store-outside-react.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+keywords: state, outside, react
+published: false
+---
+
+Jotai's state resides in React, but sometimes it would be nice
+to interact with the world outside React.
+
+## createStore
+
+[`createStore`](../core/store) provides a store interface that can be used to store your atoms. Using the store, you can access and mutate the state of your stored atoms from outside React.
```
Comment (dai-shi) [inline docs/guides/using-store-outside-react.mdx:14]: We need the extension for browsing in GitHub.
```suggestion
[`createStore`](../core/store.mdx) provides a store interface that can be used to store your atoms. Using the store, you can access and mutate the state of your stored atoms from outside React.
```

**Review by dai-shi — COMMENTED** [review]
Thanks for working on this.

It's bare minimum, but a good start.

File: docs/guides/using-store-outside-react.mdx (hunk 2 above)
Comment (ashuvssut) [reply docs/guides/using-store-outside-react.mdx]: image

Done
added `.mdx#createstore`

Comment (ashuvssut) [reply docs/guides/using-store-outside-react.mdx]: > It's bare minimum, but a good start.

Is there anything else you would like me to add here?

File: docs/guides/using-store-outside-react.mdx (hunk 1 above)
Comment (ashuvssut) [reply docs/guides/using-store-outside-react.mdx]: Updates the CodeSandbox too

image

**Comment by ashuvssut** [comment]
I can provide Edit access here. Let me know if its needed

image

**Review by dai-shi — COMMENTED** [review]
LGTM

Please run prettier.

**Comment by ashuvssut** [comment]
> LGTM
> 
> Please run prettier.

Oops my bad

done

**Review by dai-shi — APPROVED** [review]

**Pull request merged**